app/javascript/react/components/FruitShowContainer.js

```diff
@@ -0,0 +1,27 @@
+import React, { useState, useEffect } from "react"
+import FruitShow from "./FruitShow"
+
+const FruitShowContainer = (props) => {
+  const [fruit, setFruit] = useState({})
+
+  useEffect(() => {
+    const fruitId = props.match.params.id
+
+    fetch(`/api/v1/fruits/${fruitId}`)
```

Comment:
whenever we issue fetch requests we should always use the error handling.

Reply:
added error handling